File: COM_SIC_Procesa_Pagos/DataPowerRest/Status.cs
using System;

namespace COM_SIC_Procesa_Pagos.DataPowerRest
{
	[Serializable]
	public class Status
	{
		private string _type;
		private string _code;
		private string _message;
		private string _msgid;

		public string type
		{
			set{_type= value;}
			get{ return _type;}
		}
		public string code
		{
			set{_code= value;}
			get{ return _code;}
		}
		public string message
		{
			set{_message= value;}
			get{ return _message;}
		}
		public string msgid
		{
			set{_msgid= value;}
			get{ return _msgid;}
		}
		public Status()
		{

		}
	}
}

File: COM_SIC_Activaciones/DataPowerRest/Status.vb
'INC000002161718 inicio
Imports System
Public Class Status
    Private _type As String
    Private _code As String
    Private _message As String
    Private _msgid As String

    Public Property type() As String
        Get
            Return _type
        End Get
        Set(ByVal Value As String)
            _type = Value
        End Set
    End Property
    Public Property code() As String
        Get
            Return _code
        End Get
        Set(ByVal Value As String)
            _code = Value
        End Set
    End Property
    Public Property message() As String
        Get
            Return _message
        End Get
        Set(ByVal Value As String)
            _message = Value
        End Set
    End Property
    Public Property msgid() As String
        Get
            Return _msgid
        End Get
        Set(ByVal Value As String)
            _msgid = Value
        End Set
    End Property


End Class
